src/styles/blocks/_stopwatch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
```

Comment:
You can call this file just `stopwatch.scss`. No need to underline if it's a file for BEM block